tensorflow/stream_executor/dnn.h

```diff
@@ -1383,6 +1383,15 @@ class DnnSupport {
       const dnn::ConvolutionDescriptor& convolution_descriptor,
       std::vector<std::unique_ptr<dnn::ConvolveExecutionPlan>>* out_exec_plans);
 
+  virtual bool GetFusedConvolveExecutionPlans(
```

Comment:
Can you move this to `cuda_dnn.h` as well, similar to `DoConvolveXXX`? Similarly, they can be accessed via `dynamic_cast`.